controls/pcidss_4_ocp4.yml

```diff
@@ -3023,11 +3057,21 @@ controls:
         - Internal systems receive time information only from designated central time server(s).
       levels:
         - base
-      status: pending
+      status: planned
       notes: |-
-        The selected rules might need updates in order to restrict their platform applicability
-        to avoid conflicts.
+        OpenShift is configured to use a public ntppool.org which is good and reliable, but
+        ultimately is not enterprise grade.
+        The notion of central time server implies that an exclusive NTP server is deployed to
+        serve the cluster.
+
+        NTP configuration is done at the node level, except when using Precision Time Protocol
+        (PTP), for which PTP Operator can be used. PTP can only be used on bare metal deployments.
+        https://docs.openshift.com/container-platform/latest/networking/ptp/about-ptp.html
       rules: []
+      related_rules:
+        - service_chronyd_enabled
+        - var_multiple_time_servers=generic
+        - chronyd_specify_remote_server
```

Comment:
I think we need to remove `service_chronyd_enabled` and `chronyd_specify_remote_server` `var_multiple_time_servers`, since they are in the linux folder

Reply:
These are under the `related_rules` key so they shouldn't get generated in the final datastream:

> The build system adds all XCCDF rules listed under rules key in the control to the built profile. The rules listed under related_rules key are not added. Therefore, the related_rules don’t affect the generated source data stream. Also, the selections from selection key in profile file are included.

https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html#using-controls-in-profiles

These will make it easier though if/when we build out an RHCOS4 profile.

Reply:
Thanks a lot for the clarification.

Reply:
I missed that part, thanks for pointing it out!